soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -4989,12 +4983,115 @@ void CreateNaviRandoMessages() {
     }
 }
 
+CustomMessageMinimal IceTrapMessages[NUM_ICE_TRAP_MESSAGES] = {
+    { "You are a %bFOOL%w!",
+      "Du bist ein %bDUMMKOPF%w!",
+      "Tu es un %bIDIOT%w!" },
```

Comment:
i know in english the word "idiot" comes with quite a bit of baggage (see https://en.wikipedia.org/wiki/Idiot for details)

i'm not sure if it's more acceptable in french, just want to make sure the conversation is had before this gets merged (even if we already have it in current builds)

Reply:
Yeah that was my first reaction as well when @PurpleHato made this for the initial ice trap message. I can't really comment on if it's acceptable in that language, but I do have the same doubts about it honestly.

Reply:
We already had this discussion, over Idiot, Imbécile, etc
If it's too hard on the word for you, feel free to take any other randomised ice Trap message and copy paste it instead

Reply:
If you're sure it's not going to be offensive in French, I'm cool with it. Doesn't hurt to double check.

Reply:
Feel free to replace the entire sentence by
"Pauvre fou..." not a translation anymore but an adaptation that is waaay more easy, will please non French reader to not feel this too harsh

cf: LOTR Gandalf : Fly, you fools (Fuyez, pauvre fou),